docs/src/ref/build-strategies.md

```diff
@@ -58,6 +58,18 @@ The `FactoryBot.attributes_for` method constructs a Hash with the attributes
 and their values, using `initialize_with`. The `attributes_for_pair` and
 `attributes_for_list` methods work similarly as `build_pair` and `build_list`.
 
+```ruby
```

Comment:
This might fit better into `docs/src/using-factories/build-strategies.md`, since that documents at a higher level than the files under `ref/` do.